src/component.js

```diff
@@ -13,6 +13,7 @@ import { Fragment } from './create-element';
 export function Component(props, context) {
 	this.props = props;
 	this.context = context;
+	// this.constructor // When component is functional component, this is reseted to functional component
```

Comment:
Nit: Past tense of `reset` is also `reset`.

Reply:
Thank you for pointing out.
I fixed it.